src/Avalonia.Themes.Fluent/Controls/TitleBar.xaml

```diff
@@ -22,11 +22,13 @@
             <Panel x:Name="PART_Container">
               <Border x:Name="PART_Background"
                       Background="{TemplateBinding Background}"
-                      IsHitTestVisible="False"/>
+                      IsHitTestVisible="False"
+                      Win32Properties.NonClientHitTestResult="Caption" />
               <CaptionButtons x:Name="PART_CaptionButtons"
                               VerticalAlignment="Top" 
                               HorizontalAlignment="Right"
-                              Foreground="{TemplateBinding Foreground}" />
+                              Foreground="{TemplateBinding Foreground}"
+                              Win32Properties.NonClientHitTestResult="Client" />
```

Comment:
Shouldn't that be `Caption`, for example if we add spacing between the buttons?

Reply:
Yes, it should.

Reply:
Actually, no. `Border#PART_Background` already has Caption set on it. And if we add spacing between buttons, CaptionButtons itself is transparent, and won't be hit tested. But defining this property on CaptionButtons directly help to inherit it to any button inside.